config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  namespace :api do
```

Comment:
for version management, yo can use  [this](https://github.com/bploetz/versionist) gem, this way  is not that manual

Reply:
Done